public/assets/styles/annexes/mesures.css

```diff
@@ -35,3 +35,47 @@
   background-repeat: no-repeat;
   background-size: contain;
 }
+
+.mesures-par-statut h3 {
+  display: inline-block;
+
+  margin-bottom: 0;
+  padding: 0.3em 0.8em;
+
+  border-radius: 0.4em;
+  background-color: var(--fond-bleu-clair);
+
+  color: var(--bleu-anssi);
+  text-transform: uppercase;
+}
+
+.mesures-par-statut ul {
+  padding: 0;
+
+  line-height: 2em;
+}
+
+.mesures-par-statut li {
+  list-style-type: none;
+}
+
+.mesures-par-statut li::before {
```

Comment:
Ce sélecteur qui est surchargé en dessous me rend nerveux… Est-ce qu'on pourrait expliciter ce qui est propre au point bleu (vraisemblablement, le `height`, le `margin-right` et le `background-image` dans un sélecteur à part, typiquement `.mesures-par-statut li:not(.indispensable)::before` ?